rnd/autogpt_server/autogpt_server/data/model.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
```

Comment:
what is the scope of `.data.model`?

Reply:
It's used in `autogpt_server/blocks` only, you think it should go there?

Reply:
Not necessarily, it just confused me a bit because `.data` already is the `.model` of this application.